lib/config/presets/internal/group.ts

```diff
@@ -39,6 +39,7 @@ const staticGroups = {
         matchPackagePatterns: ['/node$'],
         excludePackageNames: ['calico/node'],
         commitMessageTopic: 'Node.js',
+        versioning: 'node',
```

Comment:
This causes issues with docker tags which have suffixes :/

Reply:
https://hub.docker.com/_/node

Reply:
Good catch..